.github/CHANGELOG.md

```diff
@@ -859,3 +859,4 @@
 1. [MISC] Standby Instrument stays ON if emergency power should be available, bug fixes - @2hwk (2Cas#1022 on discord)
 1. [CDU] Full +/- button functionality - @lhoenig (Lukas Hoenig)
 1. [DCDU] Fixed MSG- and MSG+ button labels - @tyler58546 (tyler58546)
+1. [EICAS] Fixed issue where CRUISE page was viewable on ground. - @patmack14 (Patrick Macken)
```

Comment:
Can you put this at the bottom of the list for version 0.8 please? This is under the version 0.4 heading.

Reply:
yes, no problem I will fix it.

Reply:
update completed